Android/app/src/main/res/values/config.xml

```diff
@@ -6,5 +6,5 @@
 
     <!-- Animation Values -->
     <item name="anim_duration" format="integer" type="integer">500</item>
-    <item name="slid_up_anim_duration" format="integer" type="integer">1000</item>
+    <item name="slide_up_anim_duration" format="integer" type="integer">1000</item>
```

Comment:
Hey Vik - I think the refactor was missed - doesn't this need to change in `Android/app/src/main/res/anim/slide_up.xml` also?

Reply:
Sorry for the stupidity!! 😞

Reply:
No problemo! 😁